sdk/lib/drivers/sound/mmebuddy/mmewrap.c

```diff
@@ -365,3 +365,85 @@ MmeGetPosition(
     return Result;
 }
 
+
```

Comment:
```suggestion
```
yes, I admit: I am nitpicking here. But just because I do know that sooner or later I will want to port your PR back ;-)